### DISCUSSION TITLE
Explicitly configured exporters in AutoConfigurationCustomizerProvider, for logs , metrics and traces but it some telemetry signals are going through other exporters

### DISCUSSION THREAD

**Discussion by shadow0wolf** [open-telemetry/opentelemetry-java-instrumentation#9920] [Q&A]
I have instrumented my java application and have written my custom Exporters for logs , metrics and traces , i have also customized OKHttpSender class to add a header on each send method invocation , but some of the telemetry requests reaching collector do not contain that header , it seems like the message is being sent using some other exporter , how to ensure that all telemetry messages use only the specified exporters.

**Comment by laurit** [comment]
It isn't really possible for us to tell what is going on without addition details or being able to reproduce your issue. Turn on collector logging and try to see whether you can figure out where this unexpected data originates from. Perhaps you are also using micrometer that configures its own exporter.
